libraries/AdaptiveExpressions/Expression.cs

```diff
@@ -104,166 +104,7 @@ public Expression(ExpressionEvaluator evaluator, params Expression[] children)
-            }
-            else
-            {
-                return Expression.MakeExpression(ExpressionType.SetPathToValue, property, Expression.ConstantExpression(value));
-            }
-        }
-
-        /// <summary>
-        /// Construct and validate an Equals expression.
-        /// </summary>
-        /// <param name="children">Child clauses.</param>
-        /// <returns>New expression.</returns>
-        public static Expression EqualsExpression(params Expression[] children)
-        => Expression.MakeExpression(ExpressionType.Equal, children);
-
-        /// <summary>
-        /// Construct and validate an And expression.
-        /// </summary>
-        /// <param name="children">Child clauses.</param>
-        /// <returns>New expression.</returns>
-        public static Expression AndExpression(params Expression[] children)
-        {
-            if (children.Count() > 1)
-            {
-                return Expression.MakeExpression(ExpressionType.And, children);
-            }
-
-            return children.Single();
-        }
-
-        /// <summary>
-        /// Construct and validate an Or expression.
-        /// </summary>
-        /// <param name="children">Child clauses.</param>
-        /// <returns>New expression.</returns>
-        public static Expression OrExpression(params Expression[] children)
-        {
-            if (children.Count() > 1)
-            {
-                return Expression.MakeExpression(ExpressionType.Or, children);
-            }
-
-            return children.Single();
-        }
-
-        /// <summary>
-        /// Construct and validate a Not expression.
-        /// </summary>
-        /// <param name="child">Child clauses.</param>
-        /// <returns>New expression.</returns>
-        public static Expression NotExpression(Expression child)
-            => Expression.MakeExpression(ExpressionType.Not, child);
-
-        /// <summary>
-        /// Construct a constant expression.
-        /// </summary>
-        /// <param name="value">Constant value.</param>
-        /// <returns>New expression.</returns>
-        public static Expression ConstantExpression(object value)
-            => new Constant(value);
-
-        /// <summary>
-        /// Construct and validate a property accessor.
-        /// </summary>
-        /// <param name="property">Property to lookup.</param>
-        /// <param name="instance">Expression to get instance that contains property or null for global state.</param>
-        /// <returns>New expression.</returns>
-        public static Expression Accessor(string property, Expression instance = null)
-            => instance == null
-            ? MakeExpression(ExpressionType.Accessor, ConstantExpression(property))
-            : MakeExpression(ExpressionType.Accessor, ConstantExpression(property), instance);
+        public static implicit operator Expression(string expression) => new ExpressionParser().Parse(expression);
 
```

Comment:
Why was this removed?  I far far prefer this signature!

Reply:
just moved it to a new class "ExpressionBuilder"